lib/article-tile/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component, PropTypes } from 'react';
 
 require('./style.css');
 
-export class ArticleTile extends Component {
+export default class ArticleTile extends Component {
```

Comment:
That link doesn't work very well - see https://github.com/numo-labs/isearch-ui/blob/0549f4370054b98f496c8cdf2fc3e43723666e18/src/components/search-results/index.js#L1-L7